frontend/src/v4/routes/viewer3D/viewer3D.component.tsx

```diff
@@ -251,18 +254,43 @@ export class Viewer3D extends PureComponent<IProps, any> {
 		}
 	}
 
+	private renderCalibrationInfoBox = (step) => {
+		if (step === 0) {
+			return (
+				<CalibrationInfoBox
+					title={formatMessage({ defaultMessage: '3D Calibration Points', id: 'infoBox.title.firstStep' })}
+					description={formatMessage({ defaultMessage: 'Select base and target point in the 3D Model to calibrate.', id: 'infoBox.description.secondStep' })}
+				/>
+			);
+		};
+		if (step === 2) {
+			return (
+				<CalibrationInfoBox
+					title={formatMessage({ defaultMessage: '2D Calibration', id: 'infoBox.title.thirdStep' })}
+					description={formatMessage({
+						defaultMessage: 'Select the floor from the model tree or select an object from the model (the system will automatically set the drawing to central depth of selected object)',
+						id: 'infoBox.description.thirdStep',
+					})}
+					hideDescriptionIcon
+				/>
+			);
+		};
+		return null;
+	};
+
 	public render() {
 		return (
 			<>
-				<ViewerContainer
-					visible={this.shouldBeVisible}
-				>
+				<ViewerContainer visible={this.shouldBeVisible}>
+					<CalibrationContext.Consumer>
+						{({ step }) => this.renderCalibrationInfoBox(step)}
```

Comment:
You could do this instead.

```suggestion
						{({ step }) => (<CalibrationInfoBox  {...stepInfoProps[step]} />) }
```